src/components/DpSelect/DpSelect.stories.mdx

```diff
@@ -0,0 +1,121 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs'
+import DpSelect from './DpSelect'
+
+<Meta
+  title="Components/Select"
+  component={DpSelect} />
+
+# DpSelect
+
+The DpSelect component is used to render a select input field with customizable options. It provides functionality for selecting an option from a dropdown list and emits a select event with the selected value.
+
+## Default usage
+
+```html
+<dp-select
+  v-model="selectModel"
+  :label="{
+          text: Translator.trans('Text')
+        }"
+  name="select_menu_name"
+  :options="selectOptions"
+  required
+  @select="updateLocalState" />
+```
+
+export const Template = (args, { argTypes }) => ({
+    props: Object.keys(argTypes),
+    components: { DpSelect },
+    template: `<dp-select v-bind="$props" />`
+});
+
+### Classes
+<Canvas>
+    <Story
+        name="Classes"
+        args={{
+            id: 'selectId',
+            label: {
+              text: 'select label'
+            },
+            classes: 'layout__item u-6-of-12 sbdocs-preview css-tmvzag'
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
```

Comment:
There is only one usage of `classes` within demosplan-core. As this is an ugly opportunity to reach into the internals of the component, i'd suggest to not document it here, rather add the following in line 61 of DpSelect:

```
    /**
     * @deprecated
     */
```

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/350/commits/1bbcb01fa0d35f8ef47d5093c513a3ca3e69604f